CONTRIBUTING.md

```diff
@@ -0,0 +1,153 @@
+# Contributing to Train
+
+We are glad you want to contribute to Train! This document will help answer common questions you may have during your first contribution.
+
+## Submitting Issues
+
+We utilize **Github Issues** for issue tracking and contributions. You can contribute in two ways:
+
+1. Reporting an issue or making a feature request [here](https://github.com/chef/train/issues/new).
+2. Adding features or fixing bugs yourself and contributing your code to Train.
+
+We ask you not to submit security concerns via Github. For details on submitting potential security issues please see <https://www.chef.io/security/>
+
+## Contribution Process
+
+We have a 3 step process for contributions:
+
+1. Commit changes to a git branch, making sure to sign-off those changes for the [Developer Certificate of Origin](#developer-certification-of-origin-dco).
+2. Create a Github Pull Request for your change, following the instructions in the pull request template.
+3. Perform a [Code Review](#code-review-process) with the project maintainers on the pull request.
+
+### Pull Request Requirements
+
+Chef Projects are built to last. We strive to ensure high quality throughout the experience. In order to ensure this, we require that all pull requests to Chef projects meet these specifications:
+
+1. **Tests:** To ensure high quality code and protect against future regressions, we require all the code in Chef Projects to have at least unit test coverage. See the [test/unit](https://github.com/chef/train/tree/master/test/unit)
+directory for the existing tests and use ```bundle exec rake test``` to run them.
+2. **Green CI Tests:** We use [Travis CI](https://travis-ci.org/) and/or [AppVeyor](https://www.appveyor.com/) CI systems to test all pull requests. We require these test runs to succeed on every pull request before being merged.
+3. **Up-to-date Documentation:**  Every code change should be reflected in an update for our [documentation](https://github.com/chef/train/tree/master/docs). We expect PRs to update the documentation with the code change.
+
+In addition to this it would be nice to include the description of the problem you are solving
+  with your change. You can use [Issue Template](https://github.com/chef/train/tree/master/ISSUE_TEMPLATE.md) in the description section
+  of the pull request.
+
+### Code Review Process
+
+Code review takes place in Github pull requests. See [this article](https://help.github.com/articles/about-pull-requests/) if you're not familiar with Github Pull Requests.
+
+Once you open a pull request, project maintainers will review your code and respond to your pull request with any feedback they might have. The process at this point is as follows:
+
+1. Two thumbs-up (:+1:) are required from project maintainers. See the master maintainers document for Train projects at <https://github.com/chef/train/blob/master/MAINTAINERS.md>.
```

Comment:
Again, no MAINTAINERS doc in the project currently. I'd recommend copying the one from the kitchen-inspec project which calls out the InSpec maintainers as the maintainers of this project, too.

Reply:
Added this =)